File: Kattis/Java/SolvingForCarrots.java
package solvingforcarrots;

import java.util.Scanner;

public class SolvingForCarrots {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int contestantsAndProblems[] = new int[2];
        int problems;
        
        for (int i = 0; i < 2; i++) {
            contestantsAndProblems[i] = input.nextInt();
        }
        input.nextLine(); 
        problems = contestantsAndProblems[1];
        
        for (int i = 0; i < contestantsAndProblems[0]; i++) {
            String description = input.nextLine();
        }
        System.out.println(problems);
    }
    
}

File: Kattis/CPlusPlus/SolvingForCarrots.cpp
#include <iostream>

int main() {
    int problems;
    int constestantsAndProblems[2];
    std::string description;

    for (int i = 0; i < 2; i++) {
        std::cin >> constestantsAndProblems[i];
    }

    problems = constestantsAndProblems[1];

    for (int j = 0; j < constestantsAndProblems[0]; j++) {
        std::cin >> description;
    }

    std::cout << problems << std::endl;

    return 0;
}